modules/dev-utils/src/main/java/org/apache/ignite/development/utils/indexreader/IgniteIndexReader.java

```diff
@@ -243,8 +249,16 @@ public IgniteIndexReader(
 
         partStores = new FilePageStore[partCnt];
 
-        for (int i = 0; i < partCnt; i++)
-            partStores[i] = filePageStoreFactory.createFilePageStoreWithEnsure(i, FLAG_DATA);
+        for (int i = 0; i < partCnt; i++) {
+            errors = new ArrayList<>();
```

Comment:
It is better to reuse the collection with cleaning, and if there are errors, then put **new ArrayList<>(errors)** to the **partStoresErrors**.